tests/bluetooth/host/id/mocks/hci_core_expects.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "mocks/hci_core.h"
+#include "mocks/hci_core_expects.h"
+
+#include <zephyr/bluetooth/buf.h>
+#include <zephyr/kernel.h>
+
+void expect_single_call_bt_unpair(uint8_t id, const bt_addr_le_t *addr)
```

Comment:
this doesn't seem used by anything?

Reply:
It's used by other UTs, but as the same implementation was used for UTs, I copied the whole file at once

Reply:
fair enough